src/Xamarin.CodeAnalysis/RequiresSuperAttributeAnalyzer.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@kzu Apache License?

Reply:
I think there's a move to MIT, we can change it too?

Reply:
Or just remove it from the source @madelinez820